include/glow/Importer/ProtobufLoader.h

```diff
@@ -105,12 +105,32 @@ class ProtobufLoader {
 
 public:
   /// \returns the node that was registered with the name \p name.
-  Node *getNodeByName(const std::string &name);
+  /// When no node has been registered for this name, the method will
+  /// either return nullptr if \p allowNullptr is true or will crash the
+  /// program otherwise. In other words, this can be used to perform
+  /// invariant checks when we know something must not be nullptr.
+  Node *getNodeByName(const std::string &name, bool allowNullptr = false) const;
```

Comment:
Maybe its better to add `getNodeByNameOrNull()`? `getNodeByName` could call it and have additional check. That way, we won't have this `/* allowNullptr */` everywhere. What do you think?

Reply:
Works for me!
Done.